client/state/google-apps-users/selectors.js

```diff
@@ -0,0 +1,25 @@
+import createSelector from 'lib/create-selector';
+
+function getGoogleAppsUsersState( state ) {
+	return state.googleAppsUsers;
+}
+
+function createGoogleAppsUsersSelector( fn ) {
+	return createSelector( fn, ( state ) => [ getGoogleAppsUsersState( state ) ] );
+}
+
+export const getByDomain = createGoogleAppsUsersSelector(
+	( state, domainName ) => getGoogleAppsUsersState( state ).items.filter( item => item.domain === domainName )
+);
+
+export const getBySite = createGoogleAppsUsersSelector(
+	( state, siteId ) => getGoogleAppsUsersState( state ).items.filter( item => item.site_id === siteId )
+);
+
+export function getAll( state ) {
```

Comment:
Yeah it is not used, but the selector acts as a library and I think it makes sense to have a `getAll` function for a library.